src/containers/LocationSearchForm/index.js

```diff
@@ -1,20 +1,32 @@
 import React from "react";
+import { useHistory } from "react-router-dom";
 
 import SubmitButton from "../../components/SubmitButton";
-import { findRestaurantsLabel, findButtonLabel } from "../../Labels";
+import {
+  findRestaurantsLabel,
+  findButtonLabel,
+  searchResultURLLabel,
+} from "../../Labels";
 
-const LocationSearchForm = () => {
+const LocationSearchForm = (props) => {
+  const { setUserInput } = props;
+  const history = useHistory();
   return (
     <div className="py-8 flex flex-col lg:justify-around lg:flex-row xl:justify-around xl:flex-row 2xl:justify-around 2xl:flex-row items-center w-1/2 mx-auto">
       <span className="text-grey">{findRestaurantsLabel}</span>
       <div className="rounded bg-white lg:w-1/2 xl:w-1/2 2xl:w-1/2 flex py-1 pr-1 pl-5 justify-between">
         <input
+          onChange={(e) => setUserInput(e.target.value)}
```

Comment:
It is a convention for some, but I'd still rather not use the `e` here, it's an event so call it `event`.